avm/res/network/express-route-circuit/main.bicep

```diff
@@ -245,6 +253,16 @@ output name string = expressRouteCircuit.name
 @description('The service key of the express route circuit.')
 output serviceKey string = expressRouteCircuit.properties.serviceKey
 
+@description('The service provider provisioning state of the express route circuit.')
+output serviceProviderProvisioningState string = expressRouteCircuit.properties.serviceProviderProvisioningState
+
+@description('The authorizations for the express route circuit.')
+output authorizations array = map(expressRouteCircuit.properties.authorizations, (authorization, id) => {
+  id: authorization.id
```

Comment:
I guess this is a resourceId? If the case, I'd recommend renaming it to `resourceId`, just to be as explicit as possible

Reply:
Yes, that would be the `resourceId` for the Authorization so that does indeed make sense to rename it to `resourceId` from just `id`. Fixed

Reply:
Much appreciated 💪  Thank you @thecmdradama